hypothesis_bio/hypothesis_bio.py

```diff
@@ -233,6 +233,7 @@ def fasta(
     sequence_source: SearchStrategy = None,
     wrap_length: Optional[int] = None,
     allow_windows_line_endings=True,
+    num_fasta: Optional[int] = None
```

Comment:
Would be better to have `min_entries` and `max_entries` instead. Similar to https://github.com/mbhall88/hypothesis-bio/blob/8d72be5b536bf947938401c313a8af1e727976c2/hypothesis_bio/hypothesis_bio.py#L526

Reply:
Done!